### PR TITLE
Blog - Deploying Infrastructure as Code with AWS CloudFormation

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
@jalletto [Preview Link](https://64cea4dcd14456687bf6213a--ephemeral-narwhal-3ac3b2.netlify.app/blog/deploying-iac-aws-cloud-formation/)

**Comment by msali123** [comment]
[Updated link](https://64cec5f72ca857796ba961fd--ephemeral-narwhal-3ac3b2.netlify.app/blog/deploying-iac-aws-cloud-formation/)

File: blog/_posts/2023-08-11-deploying-iac-aws-cloud-formation.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+editor: Mustapha Ahmad Ayodeji
+
+internal-links:
+ - deploying infrastructure
+ - deployment with aws cloudformation
+ - deploying infrastructure as code
+ - code deployment
+---
+
+**We're [Earthly](https://earthly.dev/). We simplify building software with containerization. This article shows you how to deploy Infrastructure as Code with AWS CloudFormation. However, if you are curious about getting better build times by combining ideas from Makefile and Dockerfile? [Check us out.](https://earthly.dev/)**
```
Comment (jalletto) [inline blog/_posts/2023-08-11-deploying-iac-aws-cloud-formation.md:16]:
```suggestion
 - deployment with aws cloudformation
 - deploying infrastructure as code
 - code deployment
excerpt: | 
With AWS CloudFormation, you use a declarative approach to configuring and provisioning just about any resource out of Amazon Web Services' (AWS) massive 200+ service catalog.
---

**We're [Earthly](https://earthly.dev/). We simplify building software with containerization. This article shows you how to deploy Infrastructure as Code with AWS CloudFormation. However, if you are curious about getting better build times by combining ideas from Makefile and Dockerfile? [Check us out.](https://earthly.dev/)**
```

**Review by jalletto — APPROVED** [review]

File: blog/_posts/2023-08-11-deploying-iac-aws-cloud-formation.md [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+author: Alexander Yu
+editor: Mustapha Ahmad Ayodeji
+
+internal-links:
+ - deploying infrastructure
+ - deployment with aws cloudformation
+ - deploying infrastructure as code
+ - code deployment
+excerpt: | 
+With AWS CloudFormation, you use a declarative approach to configuring and provisioning just about any resource out of Amazon Web Services' (AWS) massive 200+ service catalog.
```
Comment (jalletto) [inline blog/_posts/2023-08-11-deploying-iac-aws-cloud-formation.md:15]:
```suggestion
excerpt: | 
     With AWS CloudFormation, you use a declarative approach to configuring and provisioning just about any resource out of Amazon Web Services' (AWS) massive 200+ service catalog.
```

**Pull request merged**